addons/mail/static/src/core/common/attachment_view.js

```diff
@@ -59,4 +59,8 @@ export class AttachmentView extends Component {
             model: props.threadModel,
         });
     }
+
+    displayName() {
```

Comment:
Would make sense for it to be a getter
```suggestion
    get displayName() {
```